src/main/java/org/commcare/formplayer/beans/menus/DisplayElement.java

```diff
@@ -192,7 +192,7 @@ public String getRequiredMsg() {
         return requiredMsg;
     }
 
-    @JsonGetter(value = "group_key")
+    @JsonGetter(value = "groupKey")
```

Comment:
I couldn't tell which convention is "correct". We seem to use both, `getItemsetChoices`. I'm guessing camelcase makes more sense since that's the convention for Java and javascript (which uses the data).

But you're right, it can be removed [af803fd](https://github.com/dimagi/formplayer/pull/1511/commits/af803fdfd3eb7c29bdeb8f3640b386f7d343ba23)